saleor/graphql/account/bulk_mutations/customer_bulk_update.py

```diff
@@ -360,8 +365,14 @@ def _get_customer(cls, customer_id, external_ref):
     @classmethod
     def update_address(cls, info, instance, data, field):
         address = getattr(instance, field) or models.Address()
-        address_metadata = data.pop("metadata", [])
-        cls.update_metadata(address, address_metadata)
+        address_metadata: list[MetadataInput] = data.pop("metadata", [])
+
+        metadata_collection = cls.create_metadata_from_graphql_input(
+            address_metadata, error_field_name="metadata"
+        )
+
+        store_on_instance(metadata_collection, address, MetadataType.PUBLIC)
```

Comment:
Maybe we could import the file instead of methods? This could be more meaningful, like:
`metadata_manager.store_on_instance()`
or `store_on_instance` -> `store_metadata_on_instance`

Reply:
done